src/uk/ac/open/tabapplet/TabApplet.java

```diff
@@ -280,6 +282,23 @@ private void ditchFocus(boolean forward)
 		evalJS(js);
 	}
 
+	/**
+	 * To check if VR is paused
+	 * @param ispause boolean 
+	 **/
+	public void isPause(boolean isPause)
+	{
+		this.pauseFlag = isPause;
+		// Tell JavaScript to ditch focus for this applet id
+		String js = "isPause("+this.pauseFlag+");";
+		if(VERBOSE)
+		{
+			System.err.println("[uk.ac.open.tabapplet.TabApplet] " + focusHackId +
```

Comment:
This doesn't look like correct debugging text (copy/paste).

Reply:
Fixed